cmd/limactl/copy.go

```diff
@@ -49,13 +56,20 @@ func copyAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	arg0, err := exec.LookPath("scp")
-	if err != nil {
-		return err
+	defaultTool := Rsync
+	arg0, err := exec.LookPath(string(defaultTool))
+	if err != nil || !strings.HasSuffix(arg0, "rsync") {
```

Comment:
Why check if hte suffix has "rsync"? exec.LookupPath() can succeed and return something else?

The docs do not mention any such issue:
https://pkg.go.dev/os/exec#LookPath

Reply:
Initially, I didn't realize that` exec.LookPath` was returning the path to rsync on the host machine rather than the guest machine. I added it to debug the pipeline failures that were occurring on Alpine and some other distributions. The function successfully returned the path to rsync on the pipeline machine, but the copy operation failed because rsync wasn't available on the guest machine.